partials/content-cover-metadata.php

```diff
@@ -33,6 +33,8 @@
 									$output[] = \Pressbooks\Metadata\get_subject_from_thema( $code );
 								}
 								$book_information[ $key ] = implode( ', ', $output );
+							} elseif ( 'pb_institutions' === $key ) {
+								$book_information[ $key ] = implode( ', ', $book_information[ $key ] );
```

Comment:
Not sure how to write the case here to display the correct singular/plural form, but I think the solution involves `translate_nooped_plural`: https://developer.wordpress.org/reference/functions/translate_nooped_plural/